### PR TITLE
feat(dialog) updated LoginDialog

### PR DESCRIPTION
**Pull request by Calinteodor**
1. New LoginDialog
2. Removed unneeded check for title from InputDialog

### CODE REVIEW

File: react/features/base/dialog/components/native/InputDialog.js [hunk 1]
```diff
@@ -116,7 +112,6 @@ class InputDialog<P: Props, S: State> extends AbstractDialog<P, S> {
                     <Dialog.Input
                         autoFocus = { true }
                         onChangeText = { this._onChangeText }
-                        underlineColorAndroid = { FIELD_UNDERLINE }
```
Comment (saghul) [inline react/features/base/dialog/components/native/InputDialog.js:119]: Why did you remove this?

**Review by saghul — APPROVED** [review]
LGTM with a question.

File: react/features/base/dialog/components/native/InputDialog.js (hunk 1 above)
Comment (Calinteodor) [reply react/features/base/dialog/components/native/InputDialog.js:119]: We already have it set

 underlineColorAndroid={PlatformColor(
          `@android:color/${
            isDark ? "hint_foreground_dark" : "hint_foreground_light"
          }`
        )}
